readme.md

```diff
@@ -1,12 +1,12 @@
 # figures
 
-> Unicode symbols with Windows CMD fallbacks
+> Unicode symbols with fallbacks for older terminals
 
 [![](screenshot.png)](index.js)
 
 [*and more...*](index.js)
 
-Windows CMD only supports a [limited character set](http://en.wikipedia.org/wiki/Code_page_437).
+Terminals such as Windows Console Host only support a [limited character set](http://en.wikipedia.org/wiki/Code_page_437).
```

Comment:
I might be wrong, but I think "Windows Console Host" (the terminal) is responsible for the poor font, while "CMD" (the shell) is just a program running on that terminal. For example, Powershell can also use Windows Console Host and exhibit the same problem.

Reply:
Sure, but CMD cannot run anything else, so I think it would be useful to at least mention CMD in parens.

Reply:
Fixed :+1: